.github/ISSUE_TEMPLATE/crash.yml

```diff
@@ -0,0 +1,54 @@
+name: Crash report
+description: A hard crash of the interpreter, possibly with a core dump
+labels: ["type-crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar.
+          Exceptions unexpectedly raised from stdlib Python functions count as bugs rather than crashes.
+
+          The CPython interpreter is written in a different programming language, C.
+          A "CPython crash" is when Python itself fails, leading to a traceback in the C stack.
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: |
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```py
+        your code here
+        ```
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error messages
+      description: Enter any error messages caused by the crash, including a core dump if there is one
```

Comment:
Here it might be useful to pre-fill the textarea with
```
<details>
<summary>Error messages/core dump</summary>

[paste error here]
</details>
```

Reply:
Hmm, I did this, but I'm not sure it's great. It makes it harder to fill in the form, since now you have to manually delete the pre-filled textarea if you don't have a core dump -- if you don't, it reports you as having filled in this section. See https://github.com/AlexWaygood/cpython/issues/11.

As we discussed in #107150, I feel like most people who actually get a core dump are generally experienced enough to know how to format it properly on GitHub. So I'm inclined to revert this change.

Reply:
I think changing this to a `placeholder` rather than a `value` should resolve this, actually.